users/script.js

```diff
@@ -278,7 +278,9 @@ function showUserList(users) {
     imgElement.src = userData.picture?.url ?? DEFAULT_AVATAR;
     imgElement.classList.add('user-img-dimension');
     const pElement = document.createElement('p');
-    const node = document.createTextNode(`${userData.full_name}`);
+    const node = document.createTextNode(
+      `${userData.first_name} ${userData.last_name}`,
```

Comment:
@heyrandhir is there any reason we changed `full name` to `first + last name`?

Reply:
yes earlier in filter we only had filter for status i.e OOO , idle. that status api was doing the same thing and returning from backend. the current api is returning the same as it is stored in the users collection. i.e first name and last name so I had to do it here.